packages/jest-runner/worker.ts

```diff
@@ -25,6 +26,19 @@
 import { inspect } from "util";
 import { fuzz, FuzzerStartError } from "./fuzz";
 
+interface GeneratorFunction {
+	next: () => { value: any; done: boolean };
+	[Symbol.iterator]: () => GeneratorFunction;
+}
+
+function isGeneratorFunction(obj: any): obj is GeneratorFunction {
```

Comment:
Could this function only return a `boolean`, to reduce complexity and get rid of the extra type/interface?

Reply:
Yes, done!